README.md

```diff
@@ -54,5 +54,31 @@ will begin to drift slightly from the original design. This is expected and
 there is no expectation that the original design proposal needs to be updated
 to reflect these differences.
 
-The code and our documentation are the ultimate sources of truth. CFPs are merely
-the starting point for the implementation.
+The code and our documentation are the ultimate sources of truth. CFPs are
+the starting point for the implementation, but can also be updated as needed.
+
+# Status
+
+The status of a CFP indicates its maturity. There are four different statuses
+that a CFP can have:
+
+1. Draft: A CFP in any form (Google doc, Github issue, ect.) that has not yet
+been merged into this repository.
+
+2. Implementable: CFPs that have been approved by one committer and merged into
+this repository are given the implementable status. This means that the ideas in the CFP
+have been agreed in principle and the coding work can now be started. When merged,
+the CFP should be up to date and the relevant stakeholders should be in allignement
+even if they are still going through the practical ramifications of how to implement it.
+
+4. Updated: If there have been signifigant changes been the original CFP and the
+implementation, the CFP can be updated to reflect those changes. The new status of
+the CFP is Updated since it has changed signifigantly since it was first merged.
+
+5. Dormant: This status is given to CFP that have not or did not yet end up being
```

Comment:
Coming back to this I think this could be more confusing, it'd be clearer to keep dormant something like this. For instance if someone stumbles across a proposal and sees that it's merged with "draft", they wouldn't know if it's active or not. Dormant status makes this more clear.

Reply:
I'm also imagining that we could have a dormant CFP that then later gets resurrected, so someone picks it up, makes proposed changes, re-opens the discussion, then when agreed upon, the dormant CFP moves to implementable.

Reply:
Yea exactly. I think leaving a paper trail that someone can pick up on later if they are interested.